hw-model/src/lib.rs

```diff
@@ -137,6 +137,36 @@ pub trait HwModel {
         }
     }
 
+    fn copy_output_until_fatal_error(
+        &mut self,
+        code: u32,
```

Comment:
call this expected_fw_error (or expected_fw_error_nonfatal)?

Reply:
Fixed